repositories/utils.go

```diff
@@ -1,11 +1,27 @@
 package repositories
 
 import (
+	"context"
 	"fmt"
 
+	"github.com/checkmarble/marble-backend/models"
 	"github.com/checkmarble/marble-backend/pure_utils"
 )
 
+const postgres_audit_user_id_parameter = "custom.current_user_id"
+
+func setCurrentUserIdContext(ctx context.Context, exec Executor, userId *models.UserId) error {
+	if userId != nil {
+		_, err := exec.Exec(
+			ctx,
+			fmt.Sprintf("SELECT SET_CONFIG('%s', $1, false)", postgres_audit_user_id_parameter),
```

Comment:
`, false)` here means that the config will not be kept after the transaction. This also means that it only makes sense if called within a transaction.
We could of course let it survive for the duration of the the connection, and make sure we set if before each audited table write in a reused connection, but I'm close to 100% sure that we'll forget it and insert garbage data at some point if we do (at least, if we do without some more advanced bookkeeping - it may be possible to implement a cleanup when the connection is returned to the pool, for instance)